var/spack/repos/builtin/packages/cpio/package.py

```diff
@@ -28,4 +28,7 @@ def flag_handler(self, name, flags):
         if '%clang' in spec and name == 'cflags':
             flags.append('--rtlib=compiler-rt')
 
+        if '%fj' in spec and name == 'cflags':
+            flags.append('--rtlib=compiler-rt')
```

Comment:
I think we can simplify this a bit:
```python
if name == 'cflags':
    if '%intel' in spec:
        ...
    if '%clang' in spec or '%fj' in spec:
        ...
```

Reply:
Thank you for your kind suggestion.
I revised `if`s as your suggestion.